.devcontainer/.dockerfile

```diff
@@ -1,7 +1,22 @@
-FROM mcr.microsoft.com/vscode/devcontainers/python:3.10-bookworm
+FROM python:3.11-buster as requirements
 
-RUN pip install mkdocs \
-    && pip install mkdocs-markdownextradata-plugin \
-    && pip install mkdocs-material
+RUN pip install poetry==2.0.1 poetry-plugin-export==1.9.0
 
-EXPOSE 8000
\ No newline at end of file
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_VIRTUALENVS_CREATE=1 \
+    POETRY_CACHE_DIR=/tmp/poetry_cache
+
+WORKDIR /work
+
+COPY pyproject.toml poetry.lock ./
+
+RUN poetry export --format=requirements.txt > requirements.txt
```

Comment:
No need for this anymore, you can just do `pip install pyproject.toml` with the `pyproject.toml` I suggested below